resources/db/patches/mysql/+2~releases.sql

```diff
@@ -1,3 +1,3 @@
 # This patch will add the proc_uid column to the releases table
 
-ALTER TABLE releases ADD COLUMN proc_uid TINYINT(1) NOT NULL DEFAULT '0' COMMENT 'Has the release been UID processed?' AFTER proc_srr;
+ALTER TABLE releases ADD COLUMN proc_uid TINYINT(1) NOT NULL DEFAULT '0' COMMENT 'Has the release been UID processed?';
```

Comment:
This should be specified with an AFTER.  Since dev_srr isn't in Zed, you should select the next available column from our schema in case others have additional columns.  proc_files is the last one.